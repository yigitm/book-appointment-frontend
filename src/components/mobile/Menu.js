import React, { useState } from 'react';
import { HiMenuAlt4 } from 'react-icons/hi';
import { VscSearch } from 'react-icons/vsc';
import NavMob from './NavMob';

const Menu = () => {
  const [show, setShow] = useState(false);
  /* eslint-disable no-unused-expressions */
  const handleShow = () => {
    show ? setShow(false) : setShow(true);
  };
  /* eslint-enable no-unused-expressions */
  return (
    <div>
      <div className="flex justify-between items-center bg-orange-500 p-2">
        <button type="button" onClick={handleShow}>
          <HiMenuAlt4 size={50} className="text-white" />
        </button>
        <VscSearch
          size={40}
          className="mr-2 text-orange-500 rounded-2xl bg-white py-2"
        />
      </div>
      {show ? <NavMob /> : null}
    </div>
  );
};

export default Menu;
