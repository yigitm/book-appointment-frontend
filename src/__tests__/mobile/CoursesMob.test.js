import React from 'react';
import Menu from '../../components/mobile/Menu';

describe('Menu render', () => {
  test('Menu', () => {
    const Menu = () => <Menu />;

    expect(Menu()).toMatchSnapshot();
  });
});
