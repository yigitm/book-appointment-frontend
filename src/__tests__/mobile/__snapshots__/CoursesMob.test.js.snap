// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu render Menu 1`] = `<Menu />`;
